unix-domain-socket/src/test/scala/docs/scaladsl/UnixDomainSocketSpec.scala

```diff
@@ -31,52 +31,71 @@ class UnixDomainSocketSpec
 
   implicit val ma: ActorMaterializer = ActorMaterializer()
 
+  private val dir = Files.createTempDirectory("UnixDomainSocketSpec")
+
   "A Unix Domain Socket" should {
-    "receive what is sent" in {
+    "receive some bytes" in {
       //#binding
       val file: java.io.File = // ...
         //#binding
-        Files.createTempFile("UnixDomainSocketSpec1", ".sock").toFile
-      file.delete()
-      file.deleteOnExit()
+        dir.resolve("sock1").toFile
+
+      val received = Promise[ByteString]
+
+      val serverSideFlow = Flow[ByteString]
+        .buffer(1, OverflowStrategy.backpressure)
+        .wireTap(bytes => received.success(bytes))
 
       //#binding
       val binding: Future[UnixDomainSocket.ServerBinding] =
-        UnixDomainSocket().bindAndHandle(Flow.fromFunction(identity), file)
+        UnixDomainSocket().bindAndHandle(serverSideFlow, file)
       //#binding
 
       //#outgoingConnection
-      binding.flatMap { connection =>
+      binding.flatMap { _ => // connection
         val sendBytes = ByteString("Hello")
+        Source
+          .single(sendBytes)
+          .via(UnixDomainSocket().outgoingConnection(file))
+          .runWith(Sink.ignore)
+        //#outgoingConnection
+        received.future.map(receiveBytes => assert(receiveBytes == sendBytes))
+        //#outgoingConnection
+      }
+      //#outgoingConnection
+    }
+
+    "send and receive more ten times the size of a buffer" ignore {
+      val BufferSizeBytes = 64 * 1024
```

Comment:
This test presently fails so it is an area worth further investigation. Already discussed by #1349.